client/components/table/style.scss

```diff
@@ -10,6 +10,11 @@
 	.woocommerce-card__menu {
 		justify-self: flex-end;
 	}
+
+	&.is-placeholder {
+		animation: loading-fade 1.6s ease-in-out infinite;
```

Comment:
Do you need the styles in `analytics/style.scss` if you're also styling the placeholder here?